runners/flink/src/test/java/org/apache/beam/runners/flink/FlinkSavepointTest.java

```diff
@@ -133,6 +133,8 @@ public void testSavepointRestoreLegacy() throws Exception {
     runSavepointAndRestore(false);
   }
 
+  // TODO(ryan): make these fail when an exception is thrown (in the runner, I
+  //  think?), instead of just timing out
```

Comment:
Is this related to the metrics? Have you seen this test timing out?

Reply:
It's not directly related, but when working on metrics PVR tests, if something throws in the runner harness (I think), that manifests as a hang and timeout of this test case, without a good stack trace.

Not essential to add this, ofc, but it's not the first time I've experienced it while doing unrelated Flink work, so thought I'd try to raise awareness.